### PR TITLE
br: fix incorrect uri for the file storage

### LINKED ISSUE #48452: external_storage: `Local.URI` always return abs path
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
Parse `file://foo/bar` to external storage, and format it back via `URI` call.

### 2. What did you expect to see? (Required)
It returns `file://foo/bar`.

### 3. What did you see instead (Required)
It returns `file:///foo/bar`.

### 4. What is your TiDB version? (Required)
Current master.

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve?

Issue Number: close #48452

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by YuJuncen — APPROVED** [review]

**Review by Leavrth — APPROVED** [review]

File: br/pkg/storage/local.go [hunk 1]
```diff
@@ -140,7 +140,7 @@ func (l *LocalStorage) WalkDir(_ context.Context, opt *WalkOption, fn func(strin
 
 // URI returns the base path as an URI with a file:/// prefix.
 func (l *LocalStorage) URI() string {
-	return LocalURIPrefix + "/" + l.base
+	return LocalURIPrefix + l.base
```
Comment (hawkingrei) [inline br/pkg/storage/local.go:143]: Is it safe to use ```filepath.Join```?

**Review by hawkingrei — APPROVED** [review]

**Pull request merged**